Framework/API/src/SpectrumInfo.cpp

```diff
@@ -132,6 +132,16 @@ Kernel::V3D SpectrumInfo::position(const size_t index) const {
   return newPos / static_cast<double>(dets.size());
 }
 
+/// Returns the phi angle (in radians) for the spectrum with given index
+double SpectrumInfo::phi(const size_t index) const {
+  if (isMonitor(index))
+    throw std::logic_error("phi is not defined for monitors.");
+  Kernel::V3D pos = position(index);
+  double p, dummy;
+  pos.getSpherical(dummy, dummy, p);
```

Comment:
I'm debating whether using the same dummy arguments for `R` and `theta` is genius (avoids calculating theta) or just evil aliasing.